src/app/(home)/Table/Columns.tsx

```diff
@@ -0,0 +1,235 @@
+          </Link>
+        ) : (
+          '-'
+        )}
+      </>
+    ),
+  },
+  {
+    id: 'adminLink',
+    accessorKey: 'meta_data.admin_testing_link',
+    header: 'Admin Link',
+    cell: ({ row }) => (
+      <>
+        {row.getValue('adminLink') ? (
+          <Link
+            href={row.getValue('adminLink')}
+            target="_blank"
+            rel="noopener noreferrer"
+            className="inline-flex p-1"
+          >
+            <LinkIcon className="size-4 mx-auto cursor-pointer" />
+          </Link>
+        ) : (
+          '-'
+        )}
+      </>
+    ),
+  },
+  {
+    id: 'isEnabled',
+    accessorKey: 'meta_data.enabled',
+    header: 'Enabled',
+    cell: ({ row }) =>
+      row.getValue('isEnabled') ? (
+        <Check className="mx-auto" color="darkgreen" />
+      ) : (
+        <X className="mx-auto" color="darkred" />
+      ),
+  },
+  {
+    id: 'actions',
+    header: 'Actions',
+    enableHiding: false,
+    cell: function Cell({ row }) {
+      return (
+        <DropdownMenu>
+          <DropdownMenuTrigger asChild>
+            <Button variant="ghost" className="h-8 w-8 p-0">
+              <span className="sr-only">Open menu</span>
+              <MoreHorizontal className="h-4 w-4" />
+            </Button>
+          </DropdownMenuTrigger>
+          <DropdownMenuContent>
+            <DropdownMenuItem asChild className="cursor-pointer">
+              <Link href={`/session/edit?id=${row.original.id}`}>Edit</Link>
+            </DropdownMenuItem>
+            <DropdownMenuItem asChild className="cursor-pointer">
+              <Link href={`/session/duplicate?id=${row.original.id}`}>Duplicate</Link>
+            </DropdownMenuItem>
+            <DropdownMenuItem asChild className="cursor-pointer">
+              <Link href={`/session?id=${row.original.id}`}>View Details</Link>
+            </DropdownMenuItem>
+          </DropdownMenuContent>
+        </DropdownMenu>
+      );
+    },
+  },
+];
+
+export const displayData = (data: Session) => {
+  const basicDetails: DataSection = {
+    title: 'Basic Details',
+    data: [
+      { label: 'Platform', value: data.platform },
+      { label: 'Program', value: data.meta_data?.group },
+      { label: 'Batch', value: data.meta_data?.batch },
+      { label: 'Grade', value: data.meta_data?.grade },
+      { label: 'Session Type', value: data.type },
+      { label: 'Auth Type', value: data.auth_type },
+      { label: 'Activate Sign-Up', value: data.is_active ? 'Yes' : 'No' },
+      { label: 'Is pop up form allowed', value: data.popup_form ? 'Yes' : 'No' },
+      {
+        label: 'Number fields in pop up form',
+        value: data.meta_data?.number_of_fields_in_popup_form,
+      },
+      { label: 'Is redirection allowed', value: data.redirection ? 'Yes' : 'No' },
+      { label: 'Is Id generation allowed', value: data.id_generation ? 'Yes' : 'No' },
+      { label: 'Signup form name', value: data.meta_data?.signup_form_name ?? 'N/A' },
```

Comment:
Signup Form Name

Reply:
note that this is displayed only if signup_form is true etc .. if signup_form is false then no need to show.

infact if session type is "sign-in" or "sign-in with forgot id" .. signup_form should be false and signup_form_name option shoudln't be displayed at all. 

keep in mind for later if not this PR. make an issue in github repo